### PR TITLE
Add organisation permission models and specific create project permission

### PR DESCRIPTION
**Pull request by matthewelwell**

### CODE REVIEW

**Review by gagantrivedi — DISMISSED** [review]

**Comment by matthewelwell** [comment]
This should ideally not be merged until the FE work is complete. There is a migration which maintains the current behaviour for existing users, however, new regular organisation users will not be able to create projects without the correct permission being added or being made organisation admins.

**Comment by matthewelwell** [comment]
@gagantrivedi this is now ready for review. Note that I've added the tests in the permissions app for now as it made things simpler for now but I would like to prioritise an item in our tech debt backlog which reviews all of our tests across the application and centralises them.

File: api/organisations/managers.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.db.models import Manager
+
+from permissions.models import ORGANISATION_PERMISSION_TYPE
+
+
+class OrganisationPermissionManager(Manager):
+    def get_queryset(self):
+        return (
+            super(OrganisationPermissionManager, self)
```
Comment (gagantrivedi) [inline api/organisations/managers.py]: I think we don't need to pass any arguments to self, right? something like:  `  return super().get_queryset().filter(type=ORGANISATION_PERMISSION_TYPE)` ?

**Review by gagantrivedi — APPROVED** [review]

File: api/organisations/managers.py (hunk 1 above)
Comment (matthewelwell) [reply api/organisations/managers.py]: Yep, you're right - I'd always just gone on what PyCharm suggests but it seems like these 2 are equivalent. I'll update.

**Pull request merged**